scripts/ci/datenlord-monitor-test.sh

```diff
@@ -31,7 +30,8 @@ kind load docker-image prom/alertmanager:$ALERTMANAGER_VERSION
 kind load docker-image docker.elastic.co/elasticsearch/elasticsearch-oss:$ELASTIICSEARCH_VERSION
 kind load docker-image docker.elastic.co/kibana/kibana-oss:$KIBANA_VERSION
 kind load docker-image prom/node-exporter
-sh scripts/datenlord-monitor-deploy.sh deploy
+sudo chmod +x scripts/setup/datenlord-monitor-deploy.sh
```

Comment:
You can set all scripts to be executable locally, so there's no need to use chmod